Core/Core/Notifications/LocalNotificationsInteractor.swift

```diff
@@ -73,4 +80,70 @@ public extension NotificationManager {
             promise(isScheduled ? .success(()) : .failure(""))
         }
     }
+
+    // MARK: - Private Helpers
+
+    private func notify(
+        identifier: String,
+        title: String,
+        body: String,
+        route: String?,
+        completion: ((Error?) -> Void)? = nil
+    ) {
+        let request = UNNotificationRequest(
+            identifier: identifier,
+            title: title,
+            body: body,
+            route: route
+        )
+        notificationCenter.add(request) { error in
+            if let error = error {
+                Analytics.shared.logError(name: "Failed to schedule local notification",
+                                          reason: error.localizedDescription)
```

Comment:
Food for thought, we could rethink (in the future) sending localized strings to logger services. It segments the data too much imo.